wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -15,17 +14,6 @@ class WhoIrisSpider(BaseSpider):
         'JOBDIR': 'crawls/who_iris'
     }
 
-    def __init__(self, *args, **kwargs):
-        settings = get_project_settings()
-        years_list = kwargs.get('years_list', False)
-        id = kwargs.get('uuid', '')
-
-        self.uuid = id
-        if years_list:
-            self.years = years_list.split(',')
-        else:
-            self.years = settings['WHO_IRIS_YEARS']
-
```

Comment:
I wasn't sure from the commit message whether this change belonged in the PR or no. Does it?

Reply:
It doesn't, I just removed this to check if we were still using it. We don't. So we can leave it here and just merge it, or I can open another PR if you prefer